src/app/common/components/modals/dialog-create-or-update-wallet/wallet-form-modal.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="wrapper">
+    <div>
+        <h1 mat-dialog-title>{{ data.title }}</h1>
+        <mat-form-field appearance="fill">
+            <mat-label>Wallet Name</mat-label>
+            <input matInput [(ngModel)]="data.inputData">
+        </mat-form-field>
+        <div mat-dialog-actions>
+        <button mat-button (click)="cancel()">No Thanks</button>
```

Comment:
I've checked the description of the ticket and actually requirement was to have only buttons "Cancel" and "Save"